### ISSUE TITLE
Add comparison of different loss functions on page /notebooks/optimisation_learning.html

### ISSUE THREAD

**Issue by ddKamel** [DeepLearning-JupyterBook/deeplearning-jupyterbook.github.io#14]
I'd add a short comparison of the different loss functions.
It's true that it always depends on the problem you want to tackle. There's no right answer on which loss function to use.
But I'd still compare things like L1 to MSE loss.

For example you could add:

> In practice the choice of the loss function depends heavily on the problem. There is no real "superior" one of any kind.
> However some loss functions get used more often in general.
> Let's for example compare the L1 Loss with the MSE. 
> Since the error in MSE are squared this creates a couple of advantages you can already recognize in the plot.
> 
> - It punishes strong outliers more heavily, since the error rises exponentially. This can often lead to more stable optimization
> - The graph of MSE is smoother. Smooth gradients work mathematically really well with commonly used optimizer like gradient descent.
> - MSE often runs faster. This arises from the fact that LSE has a non-smooth point at 0
> - It creates a mean like prediction, which is often favored in ML-problems.
> 
> In reality the MSE is more commonly used than L1. This makes it not superior by any means. L1 can outshine it in some specific problems.

You could also move on and add the advantages for L1, but I believe this will do. 

EDIT: formatting, added a word
